### PR TITLE
Add a blog post on OpenID Connect and a release blog post for WildFly 26

### PR DESCRIPTION
**Pull request by fjuma**
Can be previewed here:

https://fjuma.github.io/wildfly-elytron/blog/securing-wildfly-apps-openid-connect/
http://fjuma.github.io/wildfly-elytron/blog/new-security-features-wildfly/

### CODE REVIEW

File: _posts/2021-12-15-securing-wildfly-apps-openid-connect.adoc [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+contains an https://github.com/wildfly-security-incubator/elytron-examples/blob/master/simple-webapp-oidc/src/main/webapp/WEB-INF/oidc.json[oidc.json] file and its https://github.com/wildfly-security-incubator/elytron-examples/blob/master/simple-webapp-oidc/src/main/webapp/WEB-INF/web.xml#L18-L20[web.xml] file specifies `OIDC` as the `auth-method`.
+
+Now that we've set up our Keycloak OpenID provider, we just need to deploy our application to WildFly.
+
+First, we're going to start our WildFly instance (notice that we're specifying a port offset here
+since our Keycloak instance is already exposed on port 8080):
+
+[source]
+----
+./bin/standalone.sh -Djboss.socket.binding.port-offset=10
```
Comment (Skyllarr) [inline _posts/2021-12-15-securing-wildfly-apps-openid-connect.adoc:129]: @fjuma Just a detail, but offset should be 10000 here or 10 below

Comment (fjuma) [reply _posts/2021-12-15-securing-wildfly-apps-openid-connect.adoc:129]: The one below is the port itself (so 9990 + 10).

Comment (Skyllarr) [reply _posts/2021-12-15-securing-wildfly-apps-openid-connect.adoc:129]: @fjuma My mistake. Thanks!

File: _posts/2021-12-16-new-security-features-wildfly.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+feature and an example that shows how to secure an application deployed to WildFly with
+the Keycloak OpenID provider, take a look at this
+https://wildfly-security.github.io/wildfly-elytron/blog/securing-wildfly-apps-openid-connect/[blog post].
+
+== Upcoming Filesystem Realm Enhancements
+
+We will be adding the ability to encrypt the identities that are stored in Elytron's filesystem-based
+security realm. In addition to this, we will be adding the ability to verify the integrity of these
+identities as well. For all the details on this upcoming feature, check out this https://wildfly-security.github.io/wildfly-elytron/blog/upcoming-filesystem-encryption-integrity[blog post].
+
```
Comment (Skyllarr) [inline _posts/2021-12-16-new-security-features-wildfly.adoc:33]: @fjuma 
== Upcoming client side default SSL context provider

We will be adding a Java security provider that loads Elytron client configuration and registers default SSL context for the JVM. When this provider is registered with high enough priority, client libraries that use `SSLContext.getDefault()` will obtain SSL context instantiated and configured by Elytron client. For all the details on this upcoming feature, check out this 
https://wildfly-security.github.io/wildfly-elytron/blog/client-side-jvm-wide-default-sslcontext/[blog post].

**Pull request merged**